Homework/Task2_3/Entities.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2_3
+{
```

Comment:
Move to classes to files

Reply:
Added Lecture, Training and Training Content class files, renamed Entities into PracticalLesson